.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "content_tmp/ansible"]
```

Comment:
@douglasgabriel this likely won't work when this is bundled from the main app (aka most developer workflows and the build system).

Reply:
Yeah I had a feeling that it won't work, but, at this point, I am just doing some tests about how to run the ansible inside our provider, and I was not concern on how we will serve this content. But, anyway, is there some approach to embed this kind of content in ManageIQ?

Reply:
I wouldn't wait for a solution to this, we don't have anything planned.  For now I would copy&paste the playbooks here.  You should be able to seed the roles from github with the requirements.yml.

Reply:
Hmm, got it